source/uwp/Renderer/lib/AdaptiveActionElement.cpp

```diff
@@ -23,6 +24,15 @@ namespace AdaptiveNamespace
         RETURN_IF_FAILED(UTF8ToHString(sharedModel->GetSentiment(), m_sentiment.GetAddressOf()));
 
         m_internalId = sharedModel->GetInternalId();
+        m_fallbackType = MapSharedFallbackTypeToUwp(sharedModel->GetFallbackType());
```

Comment:
>MapSharedFallbackTypeToUwp [](start = 25, length = 26)

Agree that this is better, but in general this code just casts the enums and assumes they match.